changelog/fragments/upgrade-operator-framework-api.yaml

```diff
@@ -0,0 +1,16 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Upgrade operator-framework/api version from `v0.3.13` to `0.3.18`.
```

Comment:
Perhaps add a short explanation explaining why we're bumping.

Reply:
I agree, in my opinion, the changelog is for end-users so we should just add what is relevant for them.
What problems upgrade the operator-framework/api solves? What are the changes between `v0.3.13` to `0.3.18`?